src/model/AdobeProductBean.cs

```diff
@@ -42,6 +42,12 @@ public AdobeProductBean(AdobeProduct product, string version, string icon, strin
         public string Name { get => name; set => name = value; }
         public string Desc { get => desc; set => desc = value; }
         public string Detail { get => detail; set => detail = value; }
-        public string LaunchPath { get => launchPath; set => launchPath = value; }
+        public string LaunchPath
+        {
+            get
+            {
+                return AdobePatchUtil.AdobeInstallPath + "/" + relLaunchPath;
```

Comment:
路径的组合建议考虑使用`System.IO`提供的`Path.Combine`方法。
```suggestion
                return System.IO.Path.Combine(AdobePatchUtil.AdobeInstallPath, relLaunchPath);
```